Formula/f/fedify.rb

```diff
@@ -1,26 +1,16 @@
 class Fedify < Formula
   desc "CLI toolchain for Fedify"
   homepage "https://fedify.dev/cli"
-  url "https://github.com/fedify-dev/fedify/archive/refs/tags/1.7.8.tar.gz"
-  sha256 "0e7d7281a3b6d0e9c0f204286eade8d06ac8d3d5b7c063a2f78184cba8833165"
+  url "https://github.com/fedify-dev/fedify/archive/refs/tags/1.8.3.tar.gz"
+  sha256 "d413744e48e1e62bc2de91e0a8ba4e5a6a3c3d2371509ad63d6017583389a61d"
   license "MIT"
   head "https://github.com/fedify-dev/fedify.git", branch: "main"
 
-  bottle do
-    sha256 cellar: :any_skip_relocation, arm64_sequoia: "bceac01dbad22a64f1b8ac8ef611ee08c500bd54a3bc9a35fb6a62242f1a3a73"
-    sha256 cellar: :any_skip_relocation, arm64_sonoma:  "adabb976d0fe2029526ed4a789bad21716e347e489e518e56819560b548742dc"
-    sha256 cellar: :any_skip_relocation, arm64_ventura: "e62858858e69610426575b38d9ea7c4f55ba45a4e35d8e05a805bbf1a7d33499"
-    sha256                               sonoma:        "c1b644b7bbfa2cdf0ae5545a49750654424fc58f3d81faaae927527fee1ab925"
-    sha256                               ventura:       "c273110304348a73ee8ef5fa3606ff138e20efcac8a37d8f5ec067889c1c20b9"
-    sha256 cellar: :any_skip_relocation, arm64_linux:   "b2065c1414acb605cd5a494350d324e42fee0ef82fc778ccd8bad50701ed6ca7"
-    sha256 cellar: :any_skip_relocation, x86_64_linux:  "267a44b36a709922167302235cd16b6c1a6f445335db214822fc4e1132d550d3"
-  end
-
```

Comment:
Please restore this `bottle` block; you don't need to modify it.

Reply:
Restored! Thanks!